backend/src/routes/environment.ts

```diff
@@ -164,5 +187,20 @@ export default (persister: Persister, provider: InstanceProvider): Router => {
     }
   );
 
+  router.get(
+    "/active",
+    authenticationMiddleware,
+    (req: RequestWithUser, res) => {
+      const activeEnvList = P4Environment.getActiveEnvironmentList(req.user.id);
+      if (activeEnvList === undefined) {
+        return res.status(500).json({
```

Comment:
Is this really an internal server error? what if the endpoint is called without setting up any environments?

Reply:
you're right! Even better! This cannot happen anymore, as in any case at least an empty array (from empty map of active envs) will be returned. Fixed